.rhcicd/floorist/floorplan.yaml

```diff
@@ -54,6 +54,33 @@ objects:
             et.display_name,
             bg.org_id,
             actively_used
+      - prefix: insights/notifications/emails_sent_by_bundle_app_org_id
+        query: >-
+          SELECT
```

Comment:
This query will show more emails than the real number because depending on the recipients settings and subscriptions, some Kafka messages sent to the email connector will not result in emails being sent.

Reply:
Hmmmm I see... because there might be a case where we attempt to Notify an org but all of their users might have the email notifications turned off, and even though we will register a success result no email might have been sent...

Reply:
What do you want to do with this query? I'm fine with merging it if the inaccurate data is acceptable.

Reply:
I am going to put it in draft until I talk to the Business Insights people to see if we can add a notice to the page explaining what the data represents, or if we can build data that comes from Kibana too.